### ISSUE TITLE
Accessibility review of content model documentation diagrams

### ISSUE THREAD

**Issue by swirtSJW** [department-of-veterans-affairs/va.gov-cms#12600]
## Description
Our entity relationship diagrams use Mermaid which may have some accessibility issues. We should review this to make sure any issues are surfaced.

### Testing notes
- Be sure to login using an admin user account
- Example page to review: https://prod.cms.va.gov/admin/reports/content-model/entity-diagram/node/basic_landing_page?max_depth=2  
- Elements to review:
  - Filtering
  - Interactive modal
  - Key

## Acceptance Criteria
- [x] Accessibility document model issue(s) identified
- [x] The [upstream issue](https://www.drupal.org/project/content_model_documentation/issues/3341727) updated with items identified

### Team
Please check the team(s) that will do this work.

- [x] `CMS Team`
- [ ] `Public Websites`
- [ ] `Facilities`
- [ ] `User support`

**Comment by laflannery** [comment]
@swirtSJW Because I am adding directly to the upstream issue for this particular review, that means I wouldn't need to add separate issues in the VA GitHub, correct?

**Comment by laflannery** [comment]
- The clickable elements within the content model do not have a visible focus indicator
  - I am focused on the Landing page node item - you can tell by the URL at the bottom of the screen. However, you can't tell just by looking at the screen:
![image](https://user-images.githubusercontent.com/106678594/220407941-aec0222c-6a1e-4c88-a659-468cff3184de.png)
- There is a duplicate breadcrumb:
![image](https://user-images.githubusercontent.com/106678594/220408591-6a3256ad-b1a4-4ebc-87d0-5eb10dcc8b4d.png)
- The heading for "Key" should be an H2 instead of H3.  Having this as an H3 here implies that it's relationally a child of the Breadcrumbs when really is a sibling section to the breadcrumbs
![image](https://user-images.githubusercontent.com/106678594/220409237-f55375a9-e1fc-4672-a293-2058bfe49a3b.png)
- There is a second set of markers with the same IDs are the first set. These are therefore being flagged as duplicate IDs in axe. What is the purpose of these IDs and can they be changed/removed:
![image](https://user-images.githubusercontent.com/106678594/220414219-f8582f33-6889-4b1e-abfc-928ce670080b.png)

**Comment by swirtSJW** [comment]
The breadcrumb issue is likely specific to our handling of VA breadcrumbs, but all the rest are module specific.  Well done @laflannery
